### PR TITLE
Implements roaring imports

### LINKED ISSUE #174: Roaring bitmap import support

### PR DESCRIPTION
**Pull request by yuce**
Fixes #174

### CODE REVIEW

**Comment by yuce** [comment]
ping!

File: client.go [hunk 1]
```diff
@@ -315,23 +316,31 @@ func (c *Client) ImportField(field *Field, iterator RecordIterator, options ...I
...
 			host:   node.Host,
 			port:   node.Port,
 		}
-		eg.Go(func() error {
-			return c.importNode(uri, columnsToImportRequest(field, shard, records))
-		})
+		if fast {
+			eg.Go(func() error {
+				bmp := columnsToBitmap(options.shardWidth, records)
+				return c.importRoaringBitmap(uri, field, shard, bmp)
```
[the remark this answers is not in the document]
Comment (travisturner) [reply client.go]: By that, do you mean relying on anti-entropy to sync the imported data to replicas? Is that even possible when the data is only on one replica? For example, I think with `replicas=3`, importing to only one would result in the imported data being overwritten by the replicas.

[the remark this answers is not in the document]
Comment (travisturner) [reply client.go]: Ah, I see. Thanks for clarifying. I agree with you're original comment: import to one shard and let pilosa handle replication.

[the remark this answers is not in the document]
Comment (yuce) [reply client.go]: Updated

[the remark this answers is not in the document]
Comment (jaffee) [reply client.go]: I've been trying to test this with taxi data, but currently running into a bug which I think is unrelated.

**Pull request merged**